src/configure/configurers/remoteGitHubWorkflowConfigurer.ts

```diff
@@ -139,7 +140,7 @@ export class RemoteGitHubWorkflowConfigurer extends LocalGitHubWorkflowConfigure
                             title: Messages.settingUpGithubSecrets
                         },
                         async () => {
-                            await this.githubClient.createOrUpdateGithubSecret(secretKey, secretValue);
+                            await createOrUpdateGithubSecret(secretKey, secretValue);
```

Comment:
You can convert the createOrUpdateGithubSecret() inside GitHub client to static, instead of defining it again.